### DISCUSSION TITLE
Accessing getPixel in Adafruit_GFX

### DISCUSSION THREAD

**Discussion by sigmaprojects** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#354] [Q&A] [answered]
This is more of a language question, how would I get access to the getPixel method in the Adafruit GFX lib?  I need the ability to get the "current state" of the matrix, which pixels have which colors.  

Regardless though, thank you for this project.  It got my matrix up and running great with no flicker.

**Comment by mrcodetastic** [comment, marked as answer]
That is not possible with this library as there is no 'buffer' of the pixel data once split into pulses of light sent to the physical panel.

You will need to setup your own temporary pixel buffer to store this information.

**Reply by sigmaprojects** [reply]
Thank you for letting me know!
